devchat.go

```diff
@@ -338,6 +346,27 @@ func (u *user) repl() {
 		}
 		u.term.Write([]byte(strings.Repeat("\033[A\033[2K", int(math.Ceil(float64(len([]rune(u.name+line))+2)/(float64(u.win.Width))))))) // basically, ceil(length of line divided by term width)
 
+		// Anti spam check
+		currentMessages, exists := antispamMessages[u.id]
+		if !exists {
+			currentMessages = 0
+		}
+		currentMessages += 1
+		antispamMessages[u.id] = currentMessages
```

Comment:
```suggestion
		antispamMessages[u.id]++
		time.AfterFunc(1*time.Second, func() {
			antispamMessages[u.id]--
		}
```

Reply:
Since maps give zero values when given a missing key, 349 - 355 can be shortened to antispamMessages[u.id]++